kernelci.org/content/en/docs/contrib.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Contributing Guidelines"
+date: 2022-12-08T09:21:56Z
+draft: false
+weight: 2
+---
+
+KernelCI core project is open for contributions. Contributions may consist of
+adding new build, tests and device types as well as features and bugfixes for
+KernelCI core tools.
+The best way to contribute is to send a PR to [kernelci-core](https://github.com/kernelci/kernelci-core).
+When the PR is created, [KernelCI staging](https://kernelci.org/docs/instances/staging)
+instance takes care of updating the [staging.kernelci.org branch](https://github.com/kernelci/kernelci-core/tree/staging.kernelci.org).
+In general the branch is updated every 8h and a limited set of builds and tests
+is run on it. More detailed information about the logic behind staging runs can
+be found [here](https://kernelci.org/docs/instances/staging).
```

Comment:
```suggestion
be found in the [staging instance documentation](https://kernelci.org/docs/instances/staging).
```

Reply:
I think `here` links are bad, the text should be informative regardless of any hyperlinks.  That's some web design "best practice" rule I learnt a while ago...  So I'd rather leave `the staging instance documentation`.

Reply:
Yeah, exactly (though I am not sure if you are agreeing or perhaps you read the diff the other direction).

Reply:
Ah yes sorry, my bad :coffee: